modules/cluster-logging-collector-log-forward-loki.adoc

```diff
@@ -1,8 +1,12 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/log-forwarding.adoc
+
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-collector-log-forward-loki_{context}"]
-= Forwarding logs to Loki
+= Forwarding logs to an external Loki logging system
 
-You can forward logs to an external Loki logging system in addition to, or instead of, the internal default {product-title} Elasticsearch instance.
+You can forward logs to an external Loki logging system in addition to, or instead of, an internal {product-title} Elasticsearch instance.
```

Comment:
I agree to stay neutral on which type of default log store is used. The "external Kafka" and "Amazon CloudWatch" targets already use a similar wording (for example Kafka):

```
You can forward logs to an external Kafka broker in addition to, or instead of, the default log store.
```

Reply:
I actually thought about changing to this but wasn't sure if it was accurate. Thanks guys!